### PR TITLE
[Team-BETA Version] ONCEHUB-52876: minimist and node-forge vulnerability fix

### PR DESCRIPTION
**Pull request by ayushbansal-so**
minimist, node-forge vulnerability has been fixed

### CODE REVIEW

**Comment by so-frontend-integrations** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **sumitsharma22**
You can assign the PR to them by writing `/assign @sumitsharma22` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/team/technocrats/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by SoniRani — APPROVED** [review]

**Comment by so-frontend-integrations** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[SoniRani](https://github.com/scheduleonce/knowledgeowl-angular/pull/66#pullrequestreview-925601721 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/team/technocrats/OWNERS)~~ [SoniRani]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by so-frontend-integrations** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[SoniRani](https://github.com/scheduleonce/knowledgeowl-angular/pull/66#pullrequestreview-925601721 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=scheduleonce%2Fknowledgeowl-angular).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/scheduleonce/knowledgeowl-angular/blob/team/technocrats/OWNERS)~~ [SoniRani]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
